File: problemas/ad-hoc/1794/1794.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let N = parseInt(lines.shift().trim());
let [LA, LB] = lines.shift().trim().split(' ').map((x) => parseInt(x));
let [SA, SB] = lines.shift().trim().split(' ').map((x) => parseInt(x));

if(LA <= N && N <= LB && SA <= N && N <= SB){
    console.log("possivel");
}else{
    console.log("impossivel");
}

File: problemas/ad-hoc/1794/1794.py
N = int(input().strip())
LA, LB = [int(x) for x in input().strip().split(' ')]
SA, SB = [int(x) for x in input().strip().split(' ')]

if(LA <= N <= LB and SA <= N <= SB):
    print("possivel")
else:
    print("impossivel")
